stock_release_channel/models/stock_release_channel.py

```diff
@@ -36,6 +36,21 @@ class StockReleaseChannel(models.Model):
     release_forbidden = fields.Boolean(string="Forbid to release this channel")
     sequence = fields.Integer(default=lambda self: self._default_sequence())
     color = fields.Integer()
+    warehouse_id = fields.Many2one(
+        "stock.warehouse",
+        string="Warehouse",
+        help="Warehouse for which this channel is relevant",
```

Comment:
Add an index on this as you add it by default in the search view.

Reply:
done